tests/test_playbooks/katello_smart_proxy.yml

```diff
@@ -26,6 +44,19 @@
   vars_files:
     - vars/server.yml
   tasks:
+    - include: tasks/lifecycle_environment.yml
+      vars:
+        lifecycle_environment_state: present
+        lifecycle_environment_name: "{{ item.name }}"
+        lifecycle_environment_label: "{{ item.label }}"
+        lifecycle_environment_prior: "{{ item.prior }}"
+      loop:
+        - name: Dev
+          label: dev
+          prior: Library
+        - name: Test
+          label: test
+          prior: Dev
```

Comment:
I'd think this part should be on `localhost` (and thus not recorded), as this is essentially "set-up" which is only needed when we run against a live instance, in the recorded case the searches for the LCEs will work as those are still recorded as part of the `smart_proxy` module invocations.

Reply:
Agreed. My request was to remove https://github.com/theforeman/foreman-ansible-modules/pull/1026/files#diff-6a0fba6bb0a7bef6d299c3dbccf84992c22c44143566227aa3ab2e983db0b993L29-R67

Reply:
Fixed!